components/hab/static/template_gitignore

```diff
@@ -0,0 +1,2 @@
+.DS_Store
```

Comment:
.DS_Store is not specific to Habitat and should be handled by the user in a [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

If you agree, that would make the contents of this file just one line and you probably could just inline that string in init.rs instead.